app/src/models/UserFile.ts

```diff
@@ -0,0 +1,123 @@
+import * as Sequelize from "sequelize";
+import { DataTypes, Model, Optional } from "sequelize";
+import { User, UserId } from "./User";
+
+export interface UserFileAttributes {
+  id: string;
+  userId?: string;
+  file_reference?: string;
+  data?: Buffer;
+  file_type?: string;
+  sector?: string;
+  status?: string;
+  url?: string;
+  gpc_ref_no?: string;
+  created?: Date;
+  lastUpdated?: Date;
+}
+
+export type UserFilePk = "id";
+export type UserFileId = UserFile[UserFilePk];
+export type UserFileOptionalAttributes =
+  | "userId"
+  | "file_reference"
+  | "data"
+  | "file_type"
+  | "sector"
+  | "url"
+  | "status"
+  | "gpc_ref_no"
+  | "created"
+  | "lastUpdated";
+export type UserFileCreationAttributes = Optional<
+  UserFileAttributes,
+  UserFileOptionalAttributes
+>;
+
+export class UserFile
+  extends Model<UserFileAttributes, UserFileCreationAttributes>
+  implements UserFileAttributes
+{
+  id!: string;
+  userId?: string | undefined;
+  file_reference?: string | undefined;
+  data: Buffer | undefined;
+  file_type?: string | undefined;
+  sector?: string | undefined;
+  url?: string | undefined;
+  status: string | undefined;
+  gpc_ref_no?: string | undefined;
+  created?: Date | undefined;
+  lastUpdated?: Date | undefined;
+
+  //UserFile belongs to User via id
+  user!: User;
+  getUser!: Sequelize.BelongsToGetAssociationMixin<User>;
+  setUser!: Sequelize.BelongsToSetAssociationMixin<User, UserId>;
+  createUser!: Sequelize.BelongsToCreateAssociationMixin<User>;
+
+  static initModel(sequelize: Sequelize.Sequelize): typeof UserFile {
+    return UserFile.init(
+      {
+        id: {
+          type: DataTypes.UUID,
+          allowNull: false,
+          primaryKey: true,
+          field: "id",
+        },
+        userId: {
+          type: DataTypes.UUID,
+          allowNull: true,
+          references: {
+            model: "User",
+            key: "user_id",
+          },
+          field: "user_id",
+        },
+        file_reference: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
+        data: {
+          type: DataTypes.BLOB,
+          allowNull: true,
+        },
+        file_type: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
+        sector: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
+        url: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
+        status: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
+        gpc_ref_no: {
+          type: DataTypes.STRING(255),
+          allowNull: true,
+        },
```

Comment:
Add `field: "file_reference"` and rename to fileReference (and similar for the others).